src/test/rpc/AccountSet_test.cpp

```diff
@@ -300,6 +402,7 @@ class AccountSet_test : public beast::unit_test::suite
         testSetAndResetAccountTxnID();
         testSetNoFreeze();
         testDomain();
+        testGateway();
```

Comment:
Of course @scottschurr, my apologies! Didn't realize you got pinged every time I pushed a new commit. Will update you + other reviewers when I'm all finished up.